### DISCUSSION TITLE
help w selectioncolor

### DISCUSSION THREAD

**Discussion by catalinaft** [sumatrapdfreader/sumatrapdf#4627] [Q&A]
Hi. Can the selection color be transparent with no color and 0 opacity? Can it be fixed from settings.text?

**Comment by GitHubRulesOK** [comment]
hmm yes I think so but why its less useful without some tell tale? there will be a boundary and it would be meaning less to have 0 opacity as its set at about 50% so you would need to alter the code for no transparency !

	SelectionColor = #ffffffff
same as 
	SelectionColor = #ffffff

![image](https://github.com/user-attachments/assets/c28ea67c-4fdc-4dd6-b288-f1f695e2e127)

![image](https://github.com/user-attachments/assets/15a1f35c-0f08-4ffc-98f1-6ccb91043655)
